packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -0,0 +1,1200 @@
+    return {
+      id: 'DataGrid-voidColumn',
+      width: VOID_COLUMN_WIDTH,
+    };
+  }
+
+  /**
+   * Returns the column's specified width or the default width if not width is defined.
+   * @param {Object} column Object adhering to the columnData shape.
+   * @param {Number} defaultColumnWidth Number (in px) indicating the default width to be used if a column's width is otherwise undefined.
+   */
+  static getWidthForColumn(column, defaultColumnWidth) {
+    return (column && column.width) || defaultColumnWidth;
+  }
+
+  /**
+   * Returns the combined width of every column provided.
+   * @param {Array} columns Array of Objects adhering to the columnData shape.
+   * @param {Number} defaultColumnWidth Number (in px) indicating the default width to be used if a column's width is otherwise undefined.
+   */
+  static getTotalColumnWidth(columns, defaultColumnWidth) {
+    if (!columns) {
+      return 0;
+    }
+
+    return columns.reduce((totalWidth, column) => totalWidth + DataGrid.getWidthForColumn(column, defaultColumnWidth), 0);
+  }
+
+  /**
+   * Returns the pinned columns from the given source, including the row selection column if necessary.
+   * @param {Object} props Object conforming to DataGrid's prop types.
+   */
+  static getPinnedColumns(props) {
+    const { pinnedColumns, hasSelectableRows } = props;
+
+    let updatedPinnedColumns = pinnedColumns;
+    if (hasSelectableRows) {
+      updatedPinnedColumns = [DataGrid.getRowSelectionColumn()].concat(updatedPinnedColumns);
+    }
+
+    return updatedPinnedColumns;
+  }
+
+  /**
+   * Returns the overflow columns from the given source, including the void column used for column resizing.
+   * @param {Object} props Object conforming to DataGrid's prop types.
+   */
+  static getOverflowColumns(props) {
+    const { overflowColumns, hasResizableColumns } = props;
+
+    if (hasResizableColumns) {
+      return overflowColumns.concat([DataGrid.getVoidColumn()]);
+    }
+
+    return overflowColumns;
+  }
+
+  /**
+   * Returns true if the given element matches the given selector. Includes support for IE10.
+   * @param {Element} element The element to compare against the selector.
+   * @param {String} selector The selector string to test.
+   */
+  static matchesSelector(element, selector) {
+    if (Element.prototype.msMatchesSelector) {
+      return element.msMatchesSelector(selector);
+    }
+
+    return element.matches(selector);
+  }
+
+  /**
+   * Returns the new position and offset ratio of the scrollbar given the change in width.
+   * @param {Number} scrollbarWidth The current scrollbar width.
+   * @param {Number} containerWidth The width of the container in which the scrollbar is presented.
+   * @param {Number} currentScrollbarPosition The current scrollbar position.
+   * @param {Number} delta The desired difference in position.
+   */
+  static calculateScrollbarPosition(scrollbarWidth, containerWidth, currentScrollbarPosition, delta) {
```

Comment:
To slim is file down It might makes sense to  create a ColumnUtils and/or RowUtils files? More so even for these methods that don't use `this.props`